src/Lib/Server/Auth.hs

```diff
@@ -22,9 +22,10 @@ import Servant.Generic ((:-), AsApi, AsServerT, ToServant)
 import Lib.App (App, AppError (..), Session (..))
 import Lib.Effects.Session (MonadSession (..))
 import Lib.Effects.User (MonadUser (..), User (..))
-import Lib.Util.App (maybeWithM, timedAction)
-import Lib.Util.JWT (JWTPayload (..), decodeAndVerifyJWTToken, mkJWTToken)
-import Lib.Util.Password (PasswordPlainText (..), verifyPassword)
+import Lib.App.Error (maybeWithM)
```

Comment:
Imports in this file (and probably in others) are not sorted alphabetically. You should use `stylish-haskell` on files to sort imports alphabetically.